parsers/verilog.c

```diff
@@ -737,20 +737,19 @@ static void dropEndContext (tokenInfo *const token)
 }
 
 
-static void createTag (tokenInfo *const token)
+static void createTag (tokenInfo *const token, verilogKind kind)
 {
 	tagEntryInfo tag;
-	verilogKind kind;
+
```

Comment:
(This is not a must.)
How about putting "Assert (kind >= 0);" here?

Reply:
Good point!

I left the following comment.
On some condition these lines are executed.  I cannot assert here.
I need some more code clean-up.

```
	/* FIXME: This if-clause should be removed. */
	if (kind == K_UNDEFINED || kind == K_IDENTIFIER)
	{
		verbose ("Unexpected token kind %d\n", kind);
		return;
	}
```